### PR TITLE
Veoss 590

### PR DESCRIPTION
**Pull request by oliwiaKot**

### CODE REVIEW

File: conf/logback.xml [hunk 1]
```diff
@@ -57,4 +57,8 @@
         <appender-ref ref="FILE"/>
         <appender-ref ref="STDOUT"/>
     </root>
+
+    <root level="WARN">
+        <appender-ref ref="STDOUT"/>
+    </root>
```
Comment (DevAndyroo) [inline conf/logback.xml:63]: can we revert this?

File: app/services/FinancialDataService.scala [hunk 2]
```diff
@@ -30,49 +30,34 @@ import scala.concurrent.{ExecutionContext, Future}
...
-    }.recover {
-      case _: Exception =>
-        None
-    }
-
     for {
       vatReturns <- vatReturnService.get(vrn)
-      maybeFinancialDataResponse <- getFinancialDataResponse
+      maybeFinancialDataResponse <- getFinancialData(vrn, commencementDate).recover {
+        case _: Exception => None
```
Comment (DevAndyroo) [inline app/services/FinancialDataService.scala:51]: Can we add a log line here with the exception passed in?

**Review by DevAndyroo — CHANGES_REQUESTED** [review]

File: app/services/FinancialDataService.scala (hunk 2 above)
Comment (oliwiaKot) [reply app/services/FinancialDataService.scala]: done now

File: conf/logback.xml (hunk 1 above)
Comment (oliwiaKot) [reply conf/logback.xml]: done

**Review by darbyo — APPROVED** [review]

**Review by DevAndyroo — APPROVED** [review]

**Pull request merged**